grammar.js

```diff
@@ -144,7 +163,7 @@ module.exports = grammar({
 
     tag_name: ($) => /[a-z]+[^<>{}!"'/=\s]*/,
 
-    attribute_name: ($) => token(prec(-1, /[^<>{}"'/=\s]+/)),
+    attribute_name: ($) => token(prec(-1, /[^:<>{}"'/=\s]+/)),
```

Comment:
I think this could prevent highlights on some alpine-js attributes, like:

```html
<button x-on:click="open = ! open">
  Toggle
</button>
```

We could something like

```js
/[^:<>{}"'/=][^<>{}"'/=\s]*/
```

instead so it allows `:` after the first character

Reply:
good catch, I pushed a fix for this